src/main/java/com/example/gradecalculator/entities/GradeType.java

```diff
@@ -23,7 +23,11 @@ public class GradeType {
     @JoinColumn(name = "grade_system")
     private GradeSystem gradeSystem;
 
+    public GradeType(long l, String description, GradeSystem gradeSystem) {
```

Comment:
also a constructor that has parameters but does never use those parameters

Reply:
without that there's error "Class 'GradeType' should have [public, protected] no-arg constructor"

Reply:
But the values are never used so the implementation in the test class are also useless, as it could also be solved with an empty constructor